mmdeploy/codebase/mmdet/core/anchor.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch.onnx import symbolic_helper
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+class GridPriorsTRTOp(torch.autograd.Function):
+
+    @staticmethod
+    def forward(ctx, base_anchors, feat_h, feat_w, stride_h: int,
+                stride_w: int):
+        device = base_anchors.device
+        dtype = base_anchors.dtype
+        shift_x = torch.arange(0, feat_w, device=device).to(dtype) * stride_w
+        shift_y = torch.arange(0, feat_h, device=device).to(dtype) * stride_h
+
+        def _meshgrid(x, y, row_major=True):
+            # use shape instead of len to keep tracing while exporting to onnx
+            xx = x.repeat(y.shape[0])
+            yy = y.view(-1, 1).repeat(1, x.shape[0]).view(-1)
+            if row_major:
+                return xx, yy
+            else:
+                return yy, xx
+
+        shift_xx, shift_yy = _meshgrid(shift_x, shift_y)
+        shifts = torch.stack([shift_xx, shift_yy, shift_xx, shift_yy], dim=-1)
+
+        all_anchors = base_anchors[None, :, :] + shifts[:, None, :]
+        all_anchors = all_anchors.view(-1, 4)
+        # then (0, 1), (0, 2), ...
+        return all_anchors
+
+    @staticmethod
+    @symbolic_helper.parse_args('v', 'v', 'v', 'i', 'i')
+    def symbolic(g, base_anchors, feat_h, feat_w, stride_h: int,
+                 stride_w: int):
+        # zero_h and zero_w is used to provide shape to GridPriorsTRT
+        feat_h = symbolic_helper._unsqueeze_helper(g, feat_h, [0])
+        feat_w = symbolic_helper._unsqueeze_helper(g, feat_w, [0])
+        zero_h = g.op(
+            'ConstantOfShape',
+            feat_h,
+            value_t=torch.tensor([0], dtype=torch.long),
+        )
+        zero_w = g.op(
+            'ConstantOfShape',
+            feat_w,
+            value_t=torch.tensor([0], dtype=torch.long),
+        )
+        return g.op(
+            'mmdeploy::GridPriorsTRT',
```

Comment:
could update the tensorrt ops doc: https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/ops/tensorrt.md#tensorrt-ops